Runtime/Components/ArenaWireLight.cs

```diff
@@ -36,6 +36,7 @@ protected override void ApplyRender()
                 {
                     case ArenaLightJson.TypeType.Directional:
                         light.type = LightType.Directional;
+                        gameObject.transform.LookAt(new Vector3(0,0,0));
```

Comment:
It looks like this is going to change the rotation of all directional lights from the rotation the user has stored in persist.  True, they may have set wonky lighting, but should we be changing it for them? Can you tell me more? Maybe I'm missing something.

Reply:
Directional lights in ARENA do not actually have working rotation components - "rotation" of the Directional light in ARENA is actually set by changing the position of the Directional Light, and then it always points towards origin, thereby setting direction. This is basically just replicating that ARENA behaviour in Unity.

Reply:
Ah, that makes sense! Cool.